configs/gatk_sv/config_overrides.toml

```diff
@@ -6,5 +6,5 @@
 
 # resource overrides for the GatherSampleEvidence stage
 
-[resource_overrides.GatherSampleEvidence.runtime_attr_wham]
+[resource_overrides.GatherSampleEvidence.runtime_attr_scramble_part2]
```

Comment:
Noting that this will cause a rerun of any `wham` jobs as the memory is now different. Fine if we don't use wham, just not sure.

Reply:
We do use Wham, but I think this override didn't achieve the expected result previously so it's not been used in a while. Fingers crossed this works as expected...